src/platform/Tizen/ConnectivityManagerImpl.cpp

```diff
@@ -253,13 +253,13 @@ bool ConnectivityManagerImpl::IsWiFiManagementStarted()
     return isActivated;
 }
 
-CHIP_ERROR ConnectivityManagerImpl::GetWiFiBssId(ByteSpan & value)
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiBssId(MutableByteSpan & value)
 {
     uint8_t * bssId = nullptr;
-    CHIP_ERROR err = Internal::WiFiMgr().GetBssId(bssId);
+    CHIP_ERROR err  = Internal::WiFiMgr().GetBssId(bssId);
     ReturnErrorOnFailure(err);
 
-    value = ByteSpan(bssId, 6);
+    value = MutableByteSpan(bssId, 6);
```

Comment:
This is an odd thing to do.  The expectation here is that you copy into the caller-provided buffer.

Reply:
As you said from #26155 issue, on master right now GetWiFiBssId takes a MutableByteSpan by #26088 .
Currently, this PR takes some time, so I couldn't check it. sorry about that.

Reply:
I rebased with the master branch. and it was changed to copy according to the modified code.